SSZipArchive/Common.h

```diff
@@ -0,0 +1,81 @@
+#ifndef SSZipCommon
```

Comment:
I got the file from https://github.com/ZipArchive/ZipArchive/blob/master/SSZipArchive/Common.h, it is unmodified. Will add a link to the README where I copied the files from.